src/ert/config/ert_config.py

```diff
@@ -787,31 +786,40 @@ def _create_observations(self) -> EnkfObs:
                             )
                         )
                     else:
-                        raise ValueError(f"Unknown ObservationType {obstype}")
-
-                return EnkfObs(obs_vectors, obs_time_list)
-            except IndexError as err:
-                if self.ensemble_config.refcase is not None:
-                    raise ObservationConfigError(
-                        f"{err}. The time map is set from the REFCASE keyword. Either "
-                        "the REFCASE has an incorrect/missing date, or the observation "
-                        "is given an incorrect date.",
-                        config_file=obs_config_file,
-                    ) from err
-                raise ObservationConfigError(
-                    f"{err}. The time map is set from the TIME_MAP "
-                    "keyword. Either the time map file has an "
-                    "incorrect/missing date, or the  observation is given an "
-                    "incorrect date.",
-                    config_file=obs_config_file,
-                ) from err
-
-            except ValueError as err:
-                raise ObservationConfigError(
-                    str(err),
-                    config_file=obs_config_file,
-                ) from err
-        return EnkfObs({}, obs_time_list)
+                        config_errors.append(
+                            ErrorInfo(
+                                message=f"Unknown ObservationType {obstype} for {obs_name}"
+                            ).set_context(obstype)
+                        )
+                        continue
+                except IndexError as err:
```

Comment:
`IndexError` should not catch and reraise here. It is currently catching broadly unanticipated IndexErrors. The message it is trying to ammend is here:

https://github.com/equinor/ert/blob/fe14635da3d064815071a676cb72d32a969bb92b/src/ert/config/observations.py#L393-L396

This is a relic from when the exception was raised from c and then amended in python.

Reply:
fixed